spec/README.mediawiki

```diff
@@ -1229,7 +1229,7 @@ for verification and relay to the counterparty.
 A match can be revoked by the server if a client fails to act within the
 [[#Exchange_Variables|broadcast timeout]]. A match revocation will result in
 penalties for the violating party only.
-The revoked match quantity is not added back to the orderbook in any form.
+The revoked match quantity is not added back to the order book in any form.
```

Comment:
In plain language, "order book" is used throughout the spec instead of "orderbook".